### PR TITLE
Load local configuration when configuration tests

### PR DESCRIPTION
**Pull request by ssssarah**
When running tests that initialise the BlueBrainNexus store, load store configuration from `./examples/notebooks/use-cases/prod-forge-nexus.yml`

### CODE REVIEW

**Comment by crisely09** [comment]
Would it make sense to add here the change mentioned today: use a particular path as default to load a local configuration? instead of fetching the raw file from GitHub

**Comment by ssssarah** [comment]
> Would it make sense to add here the change mentioned today: use a particular path as default to load a local configuration? instead of fetching the raw file from GitHub

This MR only deals with configurations used in tests. And it's only picking subparts of the configuration.
I think the default configuration thing should happen, but it doesn't feel relevant to this MR.

File: tests/specializations/stores/test_bluebrain_nexus.py [hunk 1]
```diff
@@ -119,16 +120,38 @@ def registered_person(person, store_metadata_value):
...
-def nexus_store(context_project_patch, metadata_context_patch):
+def nexus_store(context_project_patch, metadata_context_patch, store_config):
+
+    store_config_cp = copy.deepcopy(store_config)
+
+    for key in ["endpoint", "model", "bucket", "file_resource_mapping", "token"]:
+        if key in store_config_cp:
+            store_config_cp.pop(key)
+
+    store_config.pop("endpoint")
```
Comment (MFSY) [inline tests/specializations/stores/test_bluebrain_nexus.py:146]: WHy pop endpoint here ?

Comment (ssssarah) [reply tests/specializations/stores/test_bluebrain_nexus.py:146]: My mistake, I should remove it as it's done in the first loop.

**Review by MFSY — APPROVED** [review]

**Pull request merged**